src/lib/routes/admin-api/context.ts

```diff
@@ -136,6 +147,8 @@ export class ContextController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Context'],
+                    summary: 'Update an existing context field',
+                    description: `Endpoint that allows updating a custom context field. Used to toggle stickiness and add/remove legal values for this context field`,
```

Comment:
How is the PUT treated here? Does it replace the object wholesale, or is it more of a patch? If it works more like a patch (if any fields are ignored or unchanged if not provided), then that should be mentioned here.